### PR TITLE
Added tiers for some Control tests

### PR DESCRIPTION
**Pull request by quarckster**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit c7f994cb02131d3a505e98f0c8f0d5317c269bd3:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/control/test_compliance.py [hunk 1]
```diff
@@ -138,6 +138,7 @@ def _scan():
     logger.info("Scan finished")
 
 
+@pytest.mark.tier(3)
```
Comment (psav) [inline cfme/tests/control/test_compliance.py]: line 28 takes care of this

File: cfme/tests/control/test_actions.py [hunk 2]
```diff
@@ -313,6 +313,7 @@ def assign_policy_for_testing(vm, policy_for_testing, provider, policy_profile_n
     provider.unassign_policy_profiles(policy_profile_name)
 
 
+@pytest.mark.tier(3)
```
Comment (psav) [inline cfme/tests/control/test_actions.py]: line 93 sets these as tier 2, assuming you want to lower them to tier 3

File: cfme/tests/control/test_alerts.py [hunk 3]
```diff
@@ -183,6 +183,7 @@ def vm_crud(provider, vm_name, full_template):
     return VM.factory(vm_name, provider, template_name=full_template["name"])
 
 
+@pytest.mark.tier(3)
```
Comment (psav) [inline cfme/tests/control/test_alerts.py]: line 24 has this covered

**Pull request closed without merging**